test/e2e/network/ingressclass.go

```diff
@@ -57,7 +58,7 @@ var _ = SIGDescribe("IngressClass [Feature:Ingress]", func() {
 		}
 	})
 
-	ginkgo.It("should not set default value if no default IngressClass", func() {
+	framework.ConformanceIt("should not set default value if no default IngressClass", func() {
 		ingressClass1, err := createIngressClass(cs, "ingressclass1", false, f.UniqueName)
```

Comment:
this test also assumes the cluster did not set up its own default ingress class... this doesn't seem correct to promote to conformance as-is